### PR TITLE
Execute queries reorg

### PR DESCRIPTION
**Pull request by jstirnaman**
Merges SQL > Execute queries, InfluxQL > Execute queries into Query data > Execute queries.
Merges InfluxQL examples into pages with SQL examples.
Updates all client library class names, params, and signatures to reflect the latest releases.
Removes pandas from Use Python and adds more PyArrow examples without additional dependencies.
Adds note to recommend client libraries over Flight clients.
Miscellaneous fixes.
Adds a starter git pre-commit hook for Vale. Not sure it works correctly yet. To use it, copy the file into .git/hooks as `pre-commit` (without an extension).

- [ ] Signed the [InfluxData CLA](https://www.influxdata.com/legal/cla/)
  ([if necessary](https://github.com/influxdata/docs-v2/blob/master/CONTRIBUTING.md#sign-the-influxdata-cla))
- [ ] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/cloud-serverless/query-data/execute-queries/influxdb-v1-api.md [hunk 1]
```diff
@@ -65,22 +72,21 @@ curl --get https://cloud2.influxdata.com/query \
 ```
 {{% /code-placeholders %}}
 
-Replace the following:
+Replace the following configuration values:
 
 - {{% code-placeholder-key %}}`DATABASE_NAME`{{% /code-placeholder-key %}}:
-  Name of the database to query
+  the name of the [database (bucket)](/influxdb/cloud-serverless/admin/buckets/) to query.
```
Comment (jstirnaman) [inline content/influxdb/cloud-serverless/query-data/execute-queries/influxdb-v1-api.md:78]:
```suggestion
  the name of the [database (bucket)](/influxdb/cloud-serverless/admin/buckets/) to query
```

File: content/influxdb/cloud-dedicated/query-data/execute-queries/client-libraries/go.md [hunk 2]
```diff
@@ -0,0 +1,344 @@
...
+
+    "github.com/InfluxCommunity/influxdb3-go/influxdb3"
+    "github.com/apache/arrow/go/v12/arrow"
+)
+
+func Query() error {
+
+    // Instantiate the client.
+    client, err := influxdb3.New(influxdb3.ClientConfig{
+        Host:       "https://cluster-id.influxdb.io",
```
Comment (sanderson) [inline content/influxdb/cloud-dedicated/query-data/execute-queries/client-libraries/go.md:147]:
```suggestion
        Host:       "https://{{< influxdb/host >}}",
```

File: content/influxdb/cloud-dedicated/query-data/execute-queries/client-libraries/go.md [hunk 3]
```diff
@@ -0,0 +1,344 @@
...
+
+    "github.com/InfluxCommunity/influxdb3-go/influxdb3"
+    "github.com/apache/arrow/go/v12/arrow"
+)
+
+func InfluxQL() error {
+
+    // Instantiate the client.
+    client, err := influxdb3.New(influxdb3.ClientConfig{
+        Host:       "https://cluster-id.influxdb.io",
```
Comment (sanderson) [inline content/influxdb/cloud-dedicated/query-data/execute-queries/client-libraries/go.md:244]:
```suggestion
        Host:       "https://{{< influxdb/host >}}",
```

File: content/influxdb/cloud-dedicated/query-data/execute-queries/client-libraries/python.md [hunk 4]
```diff
@@ -1,23 +1,30 @@
 ---
-title: Use Python to query data with SQL
+title: Use Python and SQL or InfluxQL to query data
```
Comment (sanderson) [inline content/influxdb/cloud-dedicated/query-data/execute-queries/client-libraries/python.md:2]:
```suggestion
title: Use Python to query data
seotitle: Use Python and SQL or InfluxQL to query data
```

File: content/influxdb/cloud-dedicated/query-data/execute-queries/influxdb-v1-api.md [hunk 5]
```diff
@@ -43,35 +49,34 @@ Provide the following with your request:
...
 - **Query parameters:**
-  - **db**: Database to query
-  - **rp**: _(Optional)_ Retention policy to query 
+  - **db**: the database to query
+  - **rp**: Optional: the retention policy to query
   - **q**: URL-encoded InfluxQL query
 
 {{% code-placeholders "DATABASE_(NAME|TOKEN)" %}}
 ```sh
 curl --get https://cluster-id.influxdb.io/query \
```
Comment (sanderson) [inline content/influxdb/cloud-dedicated/query-data/execute-queries/influxdb-v1-api.md:58]:
```suggestion
curl --get https://{{< influxdb/host >}}/query \
```

File: content/influxdb/cloud-dedicated/reference/client-libraries/flight/csharp-flight.md [hunk 6]
```diff
@@ -18,3 +15,13 @@ weight: 201
 [Apache Arrow for C# .NET](https://github.com/apache/arrow/blob/main/csharp/README.md) integrates with C# .NET scripts and applications to query data stored in InfluxDB.
 
 For more information, see the [C# client example on GitHub](https://github.com/apache/arrow/tree/main/csharp/examples/FlightClientExample).
+
+{{% note %}}
+#### Use InfluxDB v3 client libraries
+
+We recommend using the [`influxdb3-csharp` C# client library](/influxdb/cloud-dedicated/reference/client-libraries/v3/csharp/) for integrating InfluxDB v3 with your C# application code.
+
+[InfluxDB v3 client libraries](/influxdb/cloud-dedicated/reference/client-libraries/v3/) client libraries wrap Apache Arrow Flight clients
```
Comment (sanderson) [inline content/influxdb/cloud-dedicated/reference/client-libraries/flight/csharp-flight.md:24]:
```suggestion
[InfluxDB v3 client libraries](/influxdb/cloud-dedicated/reference/client-libraries/v3/) wrap Apache Arrow Flight clients
```

File: content/influxdb/cloud-dedicated/reference/client-libraries/flight/go-flight.md [hunk 7]
```diff
@@ -7,13 +7,25 @@ menu:
...
 ---
 
 [Apache Arrow for Go](https://pkg.go.dev/github.com/apache/arrow/go/v12) integrates with Go scripts and applications to query data stored in InfluxDB.
 
+{{% note %}}
+#### Use InfluxDB v3 client libraries
+
+We recommend using the [`influxdb3-go` Go client library](/influxdb/cloud-dedicated/reference/client-libraries/v3/go/) for integrating InfluxDB v3 with your Go application code.
+
+[InfluxDB v3 client libraries](/influxdb/cloud-dedicated/reference/client-libraries/v3/) client libraries wrap Apache Arrow Flight clients
```
Comment (sanderson) [inline content/influxdb/cloud-dedicated/reference/client-libraries/flight/go-flight.md:24]:
```suggestion
[InfluxDB v3 client libraries](/influxdb/cloud-dedicated/reference/client-libraries/v3/) wrap Apache Arrow Flight clients
```

File: content/influxdb/cloud-dedicated/reference/client-libraries/flight/java-flightsql.md [hunk 8]
```diff
@@ -1,20 +1,507 @@
...
+}
+```
+
+{{% /expand %}}
+{{< /expand-wrapper >}}
+
+1. In your `<PROJECT_ROOT>/src/main/java` directory, create the `com/influxdb/examples` subdirectories for the `com.influxdb.examples` package.
+2. In the `examples` directory from the preceding step, create the `FlightQuery.java` class file.
+    You should have the following directory structure:
+    <!-- Can't make filesystem-diagram shortcode indent properly -->
```
Comment (sanderson) [inline content/influxdb/cloud-dedicated/reference/client-libraries/flight/java-flightsql.md:375]: Hugo hates shortcodes in nested lists 🤷🏻‍♂️

File: content/influxdb/cloud-serverless/get-started/query.md [hunk 9]
```diff
@@ -502,27 +504,27 @@ _If your project's virtual environment is already running, skip to step 3._
...
       token := os.Getenv("INFLUX_TOKEN")
-      database := "get-started"
-
+      
       // Instantiate the client.
-      client, err := influx.New(influx.Configs{
-        HostURL: "https://cloud2.influxdata.com",
-        AuthToken: token,
+      client, err := influxdb3.New(influxdb3.ClientConfig{
+        Host:     "https://cloud2.influxdata.com",
```
Comment (sanderson) [inline content/influxdb/cloud-serverless/get-started/query.md:518]:
```suggestion
        Host:     "https://{{< influxdb/host >}}",
```

File: content/influxdb/cloud-serverless/query-data/execute-queries/client-libraries/go.md [hunk 10]
```diff
@@ -0,0 +1,346 @@
+---
+title: Use Go and SQL or InfluxQL to query data
```
Comment (sanderson) [inline content/influxdb/cloud-serverless/query-data/execute-queries/client-libraries/go.md:2]:
```suggestion
title: Use Go to query data
seotitle: Use Go and SQL or InfluxQL to query data
```

File: content/influxdb/cloud-serverless/query-data/execute-queries/client-libraries/go.md [hunk 11]
```diff
@@ -0,0 +1,346 @@
...
+list_code_example: |
+    ```go
+    import (
+      "context"
+      "github.com/InfluxCommunity/influxdb3-go/influxdb3"
+    )
+
+    func Query() error {
+        client, err := influxdb3.New(influxdb3.ClientConfig{
+            Host:       "https://cluster-id.influxdb.io",
```
Comment (sanderson) [inline content/influxdb/cloud-serverless/query-data/execute-queries/client-libraries/go.md:28]:
```suggestion
            Host:       "https://cloud2.influxdata.com",
```

File: content/influxdb/cloud-serverless/query-data/execute-queries/client-libraries/go.md [hunk 12]
```diff
@@ -0,0 +1,346 @@
...
+
+    "github.com/InfluxCommunity/influxdb3-go/influxdb3"
+    "github.com/apache/arrow/go/v12/arrow"
+)
+
+func InfluxQL() error {
+
+    // Instantiate the client.
+    client, err := influxdb3.New(influxdb3.ClientConfig{
+        Host:       "https://cloud2.influxdata.com",
```
Comment (sanderson) [inline content/influxdb/cloud-serverless/query-data/execute-queries/client-libraries/go.md:245]:
```suggestion
        Host:       "https://{{< influxdb/host >}}",
```

File: content/influxdb/cloud-serverless/query-data/execute-queries/client-libraries/python.md [hunk 13]
```diff
@@ -1,30 +1,38 @@
 ---
-title: Use Python to query data with SQL
+title: Use Python and SQL or InfluxQL to query data
```
Comment (sanderson) [inline content/influxdb/cloud-serverless/query-data/execute-queries/client-libraries/python.md:2]:
```suggestion
title: Use Python to query data
seotitle: Use Python and SQL or InfluxQL to query data
```

File: content/influxdb/cloud-serverless/query-data/execute-queries/client-libraries/python.md [hunk 14]
```diff
@@ -218,19 +227,17 @@ The following example shows how to use Python with the `influxdb_client_3`
...
 {{% code-placeholders "(BUCKET|ORG|API)_(NAME|TOKEN)" %}}
 ```py
 # query-example.py
 
 from influxdb_client_3 import InfluxDBClient3
-import pandas
 
 # Instantiate an InfluxDBClient3 client configured for your bucket
 client = InfluxDBClient3(
     host='cloud2.influxdata.com',
```
Comment (sanderson) [inline content/influxdb/cloud-serverless/query-data/execute-queries/client-libraries/python.md:240]:
```suggestion
    host='{{< influxdb/host >}}',
```

File: content/influxdb/cloud-serverless/query-data/execute-queries/client-libraries/python.md [hunk 15]
```diff
@@ -282,34 +288,41 @@ For more information, see [`influxdb_client_3` query exceptions](/influxdb/cloud
...
+<!---- BEGIN SQL EXAMPLE --->
+{{% influxdb/custom-timestamps %}}
+{{% code-placeholders "(BUCKET|API)_(NAME|TOKEN)" %}}
 ```py
 # query-example.py
 
 from influxdb_client_3 import InfluxDBClient3
 
 client = InfluxDBClient3(
     host='cloud2.influxdata.com',
```
Comment (sanderson) [inline content/influxdb/cloud-serverless/query-data/execute-queries/client-libraries/python.md:325]:
```suggestion
    host='{{< influxdb/host >}}',
```

File: content/influxdb/cloud-serverless/query-data/execute-queries/client-libraries/python.md [hunk 16]
```diff
@@ -320,10 +333,82 @@ table = client.query(
...
+{{% code-tab-content %}}
+<!---- BEGIN INFLUXQL EXAMPLE ---->
+{{% code-placeholders "(BUCKET|API)_(NAME|TOKEN)" %}}
+```py
+# query-example.py
+
+from influxdb_client_3 import InfluxDBClient3
+
+client = InfluxDBClient3(
+    host='cluster-id.influxdb.io',
```
Comment (sanderson) [inline content/influxdb/cloud-serverless/query-data/execute-queries/client-libraries/python.md:370]:
```suggestion
    host='{{< influxdb/host >}}',
```

File: content/influxdb/cloud-serverless/reference/client-libraries/flight/go-flight.md [hunk 17]
```diff
@@ -1,19 +1,31 @@
...
 ---
 
 [Apache Arrow for Go](https://pkg.go.dev/github.com/apache/arrow/go/v12) integrates with Go scripts and applications to query data stored in InfluxDB.
 
+{{% note %}}
+#### Use InfluxDB v3 client libraries
+
+We recommend using the [`influxdb3-go` Go client library](/influxdb/cloud-serverless/reference/client-libraries/v3/go/) for integrating InfluxDB v3 with your Go application code.
+
+[InfluxDB v3 client libraries](/influxdb/cloud-serverless/reference/client-libraries/v3/) client libraries wrap Apache Arrow Flight clients
```
Comment (sanderson) [inline content/influxdb/cloud-serverless/reference/client-libraries/flight/go-flight.md:24]:
```suggestion
[InfluxDB v3 client libraries](/influxdb/cloud-serverless/reference/client-libraries/v3/) wrap Apache Arrow Flight clients
```

File: content/influxdb/cloud-serverless/reference/client-libraries/flight/java-flightsql.md [hunk 18]
```diff
@@ -1,20 +1,507 @@
...
 
 [Apache Arrow Flight SQL for Java](https://arrow.apache.org/docs/java/reference/org/apache/arrow/flight/sql/package-summary.html) integrates with Java applications to query and retrieve data from Flight database servers using RPC and SQL.
 
-<a href="https://arrow.apache.org/docs/java/reference/org/apache/arrow/flight/sql/package-summary.html" target="_blank" class="btn github">Java Flight SQL package</a>
+{{% note %}}
+#### Use InfluxDB v3 client libraries
+
+We recommend using the [`influxdb3-java` Java client library](/influxdb/cloud-serverless/reference/client-libraries/v3/java/) for integrating InfluxDB v3 with your Java application code.
+
+[InfluxDB v3 client libraries](/influxdb/cloud-serverless/reference/client-libraries/v3/) client libraries wrap Apache Arrow Flight clients
```
Comment (sanderson) [inline content/influxdb/cloud-serverless/reference/client-libraries/flight/java-flightsql.md:48]:
```suggestion
[InfluxDB v3 client libraries](/influxdb/cloud-serverless/reference/client-libraries/v3/) wrap Apache Arrow Flight clients
```

File: content/influxdb/cloud-serverless/reference/client-libraries/flight/python-flight.md [hunk 19]
```diff
@@ -49,6 +49,16 @@ list_code_example: |
 
 [Apache Arrow Python bindings](https://arrow.apache.org/docs/python/index.html) integrate with Python scripts and applications to query data stored in InfluxDB.
 
+{{% note %}}
+#### Use InfluxDB v3 client libraries
+
+We recommend using the [`influxdb3-python` Python client library](/influxdb/cloud-serverless/reference/client-libraries/v3/python/) for integrating InfluxDB v3 with your Python application code.
+
+[InfluxDB v3 client libraries](/influxdb/cloud-serverless/reference/client-libraries/v3/) client libraries wrap Apache Arrow Flight clients
```
Comment (sanderson) [inline content/influxdb/cloud-serverless/reference/client-libraries/flight/python-flight.md:57]:
```suggestion
[InfluxDB v3 client libraries](/influxdb/cloud-serverless/reference/client-libraries/v3/) wrap Apache Arrow Flight clients
```

File: content/influxdb/cloud-serverless/reference/client-libraries/v3/go.md [hunk 20]
```diff
@@ -18,6 +17,163 @@ aliases:
...
+package main
+
+import (
+    "github.com/InfluxCommunity/influxdb3-go/influxdb3"
+)
+
+func Query() error {
+
+    client, err := influxdb3.New(influxdb3.ClientConfig{
+        Host:       "https://cloud2.influxdata.com",
```
Comment (sanderson) [inline content/influxdb/cloud-serverless/reference/client-libraries/v3/go.md:84]:
```suggestion
        Host:       "https://{{< influxdb/host >}}",
```

**Review by sanderson — CHANGES_REQUESTED** [review]
Looks good. Thanks for doing this. Just some suggestions.

File: content/influxdb/cloud-dedicated/get-started/query.md [hunk 21]
```diff
@@ -723,21 +727,17 @@ _This tutorial assumes you installed Node.js and npm, and created an `influxdb_j
...
-        string? authToken = System.Environment
+        string? token = System.Environment
             .GetEnvironmentVariable("INFLUX_TOKEN");
 
         /**
           * Instantiate the InfluxDB client with credentials.
           **/
         using var client = new InfluxDBClient(
-            hostUrl, authToken: authToken, database: database);
+            "https://cluster-id.influxdb.io", token: token, database: database);
```
[the remark this answers is not in the document]
Comment (jstirnaman) [reply content/influxdb/cloud-dedicated/get-started/query.md:740]: Oh, I didn't know you merged that. I meant to go look.

File: content/influxdb/cloud-dedicated/reference/client-libraries/flight/python-flightsql-dbapi.md [hunk 22]
```diff
@@ -15,13 +15,13 @@ aliases:
 The [Python `flightsql-dbapi` Flight SQL DBAPI library](https://github.com/influxdata/flightsql-dbapi) integrates with Python applications using SQL to query data stored in an {{% cloud-name %}} database. The `flightsql-dbapi` library uses the [Flight SQL protocol](https://arrow.apache.org/docs/format/FlightSql.html) to query and retrieve data.
 
 {{% note %}}
-#### InfluxDB v3 client libraries
+#### Use InfluxDB v3 client libraries
 
 We recommend using the [`influxdb3-python` Python client library](/influxdb/cloud-dedicated/reference/client-libraries/v3/python/) for integrating InfluxDB v3 with your Python application code.
 
 [InfluxDB v3 client libraries](/influxdb/cloud-dedicated/reference/client-libraries/v3/) client libraries wrap Apache Arrow Flight clients
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/client-libraries/flight/python-flightsql-dbapi.md:22]:
```suggestion
[InfluxDB v3 client libraries](/influxdb/cloud-dedicated/reference/client-libraries/v3/) wrap Apache Arrow Flight clients
```

File: content/influxdb/cloud-dedicated/get-started/query.md (hunk 21 above)
[the remark this answers is not in the document]
Comment (sanderson) [reply content/influxdb/cloud-dedicated/get-started/query.md:740]: Oh shoot. I thought I had, but I haven't. One sec...

File: content/influxdb/cloud-serverless/reference/client-libraries/flight/python-flightsql-dbapi.md [hunk 23]
```diff
@@ -13,13 +13,13 @@ weight: 201
 The [Python `flightsql-dbapi` Flight SQL DBAPI library](https://github.com/influxdata/flightsql-dbapi) integrates with Python applications using SQL to query data stored in an {{% cloud-name %}} bucket. The `flightsql-dbapi` library uses the [Flight SQL protocol](https://arrow.apache.org/docs/format/FlightSql.html) to query and retrieve data.
 
 {{% note %}}
-#### InfluxDB v3 client libraries
+#### Use InfluxDB v3 client libraries
 
 We recommend using the [`influxdb3-python` Python client library](/influxdb/cloud-serverless/reference/client-libraries/v3/python/) for integrating InfluxDB v3 with your Python application code.
 
 [InfluxDB v3 client libraries](/influxdb/cloud-serverless/reference/client-libraries/v3/) client libraries wrap Apache Arrow Flight clients
```
Comment (jstirnaman) [inline content/influxdb/cloud-serverless/reference/client-libraries/flight/python-flightsql-dbapi.md:20]:
```suggestion
[InfluxDB v3 client libraries](/influxdb/cloud-serverless/reference/client-libraries/v3/) wrap Apache Arrow Flight clients
```

**Pull request merged**